source/dev/contribution_guide.rst

```diff
@@ -20,13 +20,21 @@ How to Contribute
    up and running
 #. Get familiar with our Git/GitHub workflow (see our :doc:`../user/git_guide` for more
    details)
+#. If you have an idea for a feature, create first a feature request.
+   (more details on `How to create a Feature Request`_)
 #. Does your feature require external data? (see the `Data Intake Procedure`_
    section)
 #. Develop your feature (go to the `Develop your Feature`_ section)
 #. Format your code (as explained in the `Format your Code`_ section)
 #. Document your code (see the `Document your Code`_ section)
 #. Make a pull request (see our :doc:`../user/git_guide` page)
 
+How to create a Feature Request
++++++++++++++++++++++++++++++++
+
+Navigate to the most fitting repository that your feature may belong to. Click on **Issues** and then **New issue**.
+You should have an options of templates to use. Select the **Feature request** template. Fill out
```

Comment:
Consider removing `You should have an options of templates to use.` - I would say that is implied by the following sentence.